CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 - Backport changes from fork
 - Adjustments due to backporting
 - Refactoring: answer options in command.py
+- Python action: Make SYSTEMS and CONST directly available
+- Python action: Automatix variables are now available as `VARS`, replacing `a_vars` (now deprecated)
```

Comment:
Guess we have to keep this until we release 3.0.0, thanks to semantic versioning